packages/amplication-server/src/core/entity/constants.ts

```diff
@@ -58,6 +58,12 @@ export const SYSTEM_DATA_TYPES: Set<EnumDataType> = new Set([
   EnumDataType.Roles,
 ]);
 
+export const MANIPULABLE_SYSTEM_DATA_TYPES: Set<EnumDataType> = new Set([
```

Comment:
@iamshabell my mistake, system data types are id and roles

Reply:
@abrl91 so i should remove the `Password`?

Reply:
yes